.editorconfig

```diff
@@ -0,0 +1,3 @@
+[*]
```

Comment:
about husky:

install as devDependencies `husky`

and add e.g. such key in package.json:
```
  "husky": {
    "hooks": {
      "pre-push": "yarn lint"
    }
  }
  ```

Reply:
idk, but it doesn't work :(